cli/vm/vm.go

```diff
@@ -27,10 +28,13 @@ func startVMPrompt(ctx *cli.Context) error {
 	if err := cmdargs.EnsureNone(ctx); err != nil {
 		return err
 	}
-
-	cfg, err := options.GetConfigFromContext(ctx)
-	if err != nil {
-		return cli.NewExitError(err, 1)
+	var err error
+	cfg := config.VM
+	if ctx.String("config-file") != "" || ctx.String("config-path") != "" {
+		cfg, err = options.GetConfigFromContext(ctx)
+		if err != nil {
+			return cli.NewExitError(err, 1)
+		}
```

Comment:
Remove the condition below, it's not needed anymore since you have a default designated VM config with InMemory DB included.